### PR TITLE
Added stacktrace to ecto telemetry attributes

### PR DESCRIPTION
**Pull request by antonsatin**
With latest version of ecto it's possible to see what code called query. This adds stacktrace to ecto telemetry attributes if it's present.

### CODE REVIEW

**Comment by sauron-droid** [comment]
Thanks for your pull request. Before we can look at it, you'll need to add a 'DCO signoff' to your commits.

:memo: **Please follow instructions in the [contributing guide](https://github.com/mindvalley/mv-opentelemetry/blob/master/CONTRIBUTING.md) to update your commits with the DCO**

Full details of the Developer Certificate of Origin can be found at [developercertificate.org](https://developercertificate.org/).

**The list of commits missing DCO signoff**:

- [754cc96](https://github.com/mindvalley/mv-opentelemetry/commits/754cc962271c77e5a3444d33962442f904e86772) Added stacktrace to ecto telemetry attributes

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
**Once this PR has been reviewed and has the lgtm label**, please ask for approval from [antonsatin](https://github.com/antonsatin). For more information see [the Kubernetes Code Review Process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process).

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

File: lib/mv_opentelemetry/ecto.ex
Comment (antonsatin) [inline lib/mv_opentelemetry/ecto.ex:63]: I got 2 questions @maciej-szlosarczyk 

1. Is the attribute name ok or should it be something else?
2. I checked the spec - attribute can be a list, so I turned stacktrace into list of strings. Is that correct or should it be something else as well (e.g. just one string)?

Comment (maciej-szlosarczyk) [reply lib/mv_opentelemetry/ecto.ex:63]: 1. When it is an error, you'd use "error.stacktrace" (note the namespace). There's also this: https://github.com/open-telemetry/semantic-conventions/blob/main/docs/general/attributes.md#source-code-attributes which might be nicer. Since I ran into this in the past, I'd namespace it to "ecto.stacktrace", so that if there's some other stacktrace it won't conflict with each other.
2. I _think_ it won't work - the support for lists/arrays in attributes was pretty poor, I had to manually convert lists into JSON arrays for Honeycomb. Can you instead make it a long string of strings?

**Comment by sauron-droid** [comment]
PR needs rebase.

<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: test/mv_opentelemetry/ecto_test.exs
Comment (maciej-szlosarczyk) [inline test/mv_opentelemetry/ecto_test.exs:49]: Can we make this test a little bit more detailed? `""` is also a binary. 🤔

**Review by maciej-szlosarczyk — APPROVED** [review]

**Comment by sauron-droid** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/mindvalley/mv-opentelemetry/pull/60#pullrequestreview-1717297202" title="Approved">maciej-szlosarczyk</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=mindvalley%2Fmv-opentelemetry).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/mindvalley/mv-opentelemetry/blob/main/OWNERS)~~ [maciej-szlosarczyk]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by sauron-droid** [comment]
LGTM label has been added.  <details>Git tree hash: 1293667eefaaa09b918b385d4a4bdec524f77d43</details>

**Comment by maciej-szlosarczyk** [comment]
It's approved, but I don't think we should ship this to prod for long 🤔

File: test/mv_opentelemetry/ecto_test.exs
Comment (antonsatin) [reply test/mv_opentelemetry/ecto_test.exs:49]: I don't think checking content adds value here because content of the binary is coming from ecto itself so we have no control over it

**Pull request merged**
